addons/mass_mailing/models/utm_campaign.py

```diff
@@ -66,8 +66,8 @@ def _compute_mailing_mail_count(self):
             mapped_data = dict()
             ab_testing_mapped_data = dict()
         for campaign in self:
-            campaign.mailing_mail_count = sum(mapped_data.get(campaign.id, []))
-            campaign.ab_testing_mailings_count = sum(ab_testing_mapped_data.get(campaign.id, []))
+            campaign.mailing_mail_count = sum(mapped_data.get(campaign._origin.id, []))
+            campaign.ab_testing_mailings_count = sum(ab_testing_mapped_data.get(campaign._origin.id, []))
```

Comment:
This looks a bit strange, what issue is this solving exactly?

Reply:
This is marked as resolved but it's still here?

Reply:
@awa-odoo , sorry, the comment was pending.

Reply:
I see what you mean.

In that case, we usually do `campaign._origin.id or campaign.id`.

That way it fallbacks to the original record's ID if "_origin" is missing.

(To adapt for both this and the sms counterpart).